cmd/root_test.go

```diff
@@ -69,6 +69,9 @@ func TestMain(m *testing.M) {
 	}
 	lab.Init(host, u.Username, token, false)
 
+	// Make "origin" the default remote for testing
+	forkedFromRemote = "origin"
+	forkRemote = "origin"
```

Comment:
This seems like it would mask failure cases. `origin` is the first remote listed in https://github.com/zaquestion/lab/blob/master/testdata/test.git/config which will be used in any tests that run with `copyTestRepo()`

Although this line seems to be a copypaste error, the branch in this case should be `master` https://github.com/zaquestion/lab/blob/master/testdata/test.git/config#L14 

Still I'd expect the logic you have above to detect `origin` as the correct remote to use, by seeing it as the first remote in this list.

Reply:
Oh actually I guess these lines would only apply to tests that _don't_ run using the test binary. So this is probably fine

Reply:
The issue is that the tests that failed are those that run the cmd.Run() directly, without calling `exec.Command(labBinaryPath, args...).`, with that, the `root.go:Execute()` is just bypassed and both `forkFromRemote` and `forkRemote` are left empty. Before the patch both vars were getting set to `origin` because they were being assigned to that in compile time. I should re-word the patch description to point to what I just have said.

Reply:
> Oh actually I guess these lines would only apply to tests that don't run using the test binary. So this is probably fine

Yes, exactly :). I didn't notice your message before I sent mine.
I'll re-word the commit log to point it.

Reply:
Sorry to make you do another thing, but can you expand the [code] comment to indicate that as well? After that I think this is ready to come in

Reply:
No worries at all :). I just pushed the new patch with updated comment.